src/sidebar/store/modules/sidebar-panels.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const util = require('../util');
+
+function init() {
+  return {
+    /*
+     * The `panelName` of the currently-active sidebar panel.
+     * Only one `panelName` may be active at a time, but it is valid (though not
+     * the standard use case) for multiple `SidebarPanel` components to share
+     * the same `panelName`—`panelName` is not intended as a unique ID/key.
+     *
+     * e.g. If `activeSidebarPanel` were `foobar`, all `SidebarPanel` components
+     * with `panelName` of `foobar` would be active, and thus visible.
+     *
+     */
+    activeSidebarPanel: null,
```

Comment:
So the way this works is if I hope the help panel (I realize it's not converted yet) then it will close any active one first, then open that one up?

Reply:
Can this be `activePanelName`

I think its clear its a sidebar panel from the namespace.

Reply:
Good call. Changing to `activePanelName`.